index.js

```diff
@@ -2648,6 +2650,15 @@ RemoteObjectTemplate.bindDecorators = function (objectTemplate) {
             objectTemplate._setupProperty(targetKey, props, undefined, defineProperties);
             Object.defineProperties(target, defineProperties);
         };
+
+        function applyRuleSet(prop, ruleSet) {
+            if (prop instanceof Array && prop.length > 0 && ruleSet instanceof Array && ruleSet.length > 0) {
+                return prop.some(r=> ruleSet.indexOf(r) >= 0);
```

Comment:
does this HAVE to be nested within the anonymous function? Can you take this out of there?

Reply:
@sshankar19 - I want to make this function private, we are following this pattern in framework. Probably, we can make these functions private if we move to typescript

Reply:
@sshankar19 - moved the function out of function scope.